### ISSUE TITLE
Click multi commands and Bash completion

### ISSUE THREAD

**Issue by ermingol23** [pallets/click#1613]
### Expected Behavior
I followed the example in Click documentation for implementing [custom multi commands](https://click.palletsprojects.com/en/7.x/commands/#custom-multi-commands) and then I want to combine this with Bash completion support as described [here](https://click.palletsprojects.com/en/7.x/bashcomplete/). When using TAB to complete Bash command line I expect to see a list of commands I can use, i.e.

```bash
$ mycommand <press TAB twice>
<list of possible commands>
$ mycommand 
```

### Actual Behavior

```bash
$ mycommand <press TAB twice>
$ mycommand cli
```

### Environment

* Python version: 3.8
* Click version: 7.1.1

**Comment by kx-chen** [comment]
Still an issue in the new autocomplete system (https://github.com/pallets/click/pull/1622).

**Comment by davidism** [comment]
This was fixed as part of #1622, specifically 00d5dcc9ede551dc04b20010ba0e65ca4fdbe1ff.
